pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +209,80 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_tang() {
+        return with_keydesc(client, pool, (keydesc, keydesc_set) => {
+            dialog_open({
+                Title: _("Add Tang keyserver"),
+                Fields: [
+                    TextInput("tang_url", _("Keyserver address"),
+                              {
+                                  validate: validate_url
+                              }),
+                    Skip("medskip",
+                         {
+                             visible: () => !keydesc_set
+                         }),
+                    PassInput("passphrase", _("Disk passphrase"),
+                              {
+                                  visible: () => !keydesc_set,
+                                  validate: val => !val.length && _("Passphrase cannot be empty"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230525-101055-c79bf30c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)